index.js

```diff
@@ -92,19 +92,24 @@ exports.init = function (sbot, config) {
               timer = setTimeout(again, 1000*Math.random())
             }
             //this plugin might be enabled, but a portal might not be set.
-            if(!portal) return reconnect()
+            if(!portal) {
+              reconnect()
+              return
+            }
```

Comment:
Replacing `return reconnect()` with `reconnect(); return` because reconnect() doesn't return anything, so to make the intent clear, we just want to call reconnect() *and then* return as a mechanism to bail out the control flow.